src/guides/v2.2/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+group: cloud-guide
+title: Scaled architecture
+functional_areas:
+  - Cloud
+---
+
+The Cloud infrastructure scales according to your resource needs to achieve greater efficiency. The {{site.data.var.ece}} monitors your applications and automatically adjusts capacity to maintain steady, predictable performance. Converting to this architecture helps to mitigate problems, such as latency, large spikes in traffic, or running out of storage.
```

Comment:
` monitors your applications and automatically adjusts capacity`
Currently, we do not have autoscaling :(

Reply:
Where is your magic wand, BaDos?